app/Models/Application.php

```
@@ -1555,40 +1555,206 @@ protected function buildGitCheckoutCommand($target): string
                 if ($value) {
-                    return trim($value);
+                    return $this->parseWatchPaths($value);
                 }
             }
         );
     }
 
+    public function matchWatchPaths(Collection $modified_files, ?Collection $watch_paths): Collection
+    {
+        return self::matchPaths($modified_files, $watch_paths);
+    }
```

PHPMD (2.15.0)

The parameter $modified_files is not named in camelCase. (undefined)

(CamelCaseParameterName)